packages/blade/src/components/Chip/Chip.tsx

```diff
@@ -39,7 +39,7 @@ type OnChange = ({
 }) => void;
 
 const _Chip: React.ForwardRefRenderFunction<BladeElementRef, ChipProps> = (
-  { isDisabled, value, children, icon: Icon, color, testID, _motionMeta, ...rest },
+  { isDisabled, value, children, icon: Icon, color, testID, _motionMeta, width, ...rest },
```

Comment:
we should expose `minWidth`, `maxWidth` as well right if we're exposing `width`?